docs/developers/protocol/hermez-protocol/fee-table.md

```diff
@@ -4,532 +4,537 @@
 
 Fees in L2 transaction apply a factor encoded by an index, as shown in the table above and the formula below:
 
-$0 \text{ if } i = 0$
+$0 \quad \text{ if } i = 0$
 
-$10^{-24 + \frac{i}{2}} \text{if } 1 \leq i \leq 32$
+$2^{-60 + i·(\frac{-8-(-60)}{32})} \quad \text{if} \quad 1 \leq i \leq 31$
 
-$10^{-8 + \frac{8}{192} (i - 32) } \text{if } 33 \leq i \leq 223$
+$2^{-8 + (i-32)·(\frac{0-(-8)}{160})} \quad \text{if} \quad 32 \leq i \leq 191$
 
-$10^{i - 224} \text{ if } 224 \leq i \leq 255$
+$2^{(i-192)·(\frac{0-63}{63})} \quad \text{if} \quad 192 \leq i \leq 255$
```

Comment:
I think this part is wrong.  `\frac{0-63}/{63}` is -1, but from the table below, it should be +1. Also, It's a bit strange to express the fraction like that, instead of its simplified form.

Reply:
Yes, it is wrong! Good catch xD

Regarding the fraction expression, I wanted to express exponent as: `maxExponent` - `minExponent`/`numSteps`, I think it is more clear in this way.
Note: this comment applies for the same comments regarding exponent expression